_content/setup-wallets-commandline.md

```diff
@@ -0,0 +1,141 @@
+## Interledger Protocol (ILP) TestNet: Getting Started
+
+This tutorial describes how to use the command line and test XRP to:
```

Comment:
I tend to prefer the wording "programmatic" or "using an API" since while this tutorial interacts with the API using `curl` on the command-line, what it's really demonstrating is how to interact with Interledger programmatically, and developers may want to adapt these API calls in their own language if they're building an app.